### PR TITLE
Fix: Dev-9864 Duplicate Dropdown label in Dashboard

### PR DESCRIPTION
**Pull request by Matt-DiPaolo**
## Fix: Dev-9864
https://websupport.cdc.gov/browse/DEV-9864

Removed the duplicate filter labels from Nested Dropdowns and Multi-Selects

## Testing Steps 

2 Scenarios

Scenario 1: Upload [dev-9864-Dashboard-Filters.json](https://github.com/user-attachments/files/17837859/dev-9864-Dashboard-Filters.json) and navigate to the Dashboard Preview

Expected: There are 3 Dashboard filters labeled with their style name and there is only one label for each

Scenario 2: Upload [dev-9864-Visualization-Filters.json](https://github.com/user-attachments/files/17837900/dev-9864-Visualization-Filters.json) and navigate to the Dashboard Preview

Expected: There are 3 Visualization filters labeled with their style name and there is only one label for each

## Self Review

- I have added testing steps for reviewers
- My changes generate no new warnings
- New and existing unit tests are passing

### CODE REVIEW

**Review by adamdoe — APPROVED** [review]

File: packages/dashboard/src/components/DashboardFilters/DashboardFilters.tsx [hunk 1]
```diff
@@ -99,55 +99,60 @@ const DashboardFilters: React.FC<DashboardFilterProps> = ({
...
+                <>
+                  <select
+                    id={`filter-${filterIndex}`}
+                    className='cove-form-select'
+                    data-index='0'
+                    value={loading ? 'Loading...' : filter.queuedActive || filter.active}
+                    onChange={val => {
+                      handleOnChange(filterIndex, val.target.value)
+                    }}
+                    disabled={loading ? true : values.length === 1 && !nullVal(filter)}
```
Comment (joshlacey) [inline packages/dashboard/src/components/DashboardFilters/DashboardFilters.tsx:141]: @Matt-DiPaolo isDisabled was being used here, why did you remove it?
